cluster/images/provider-gcp-controller/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM BASEIMAGE
```

Comment:
I want to circle back and clean this up as part of https://github.com/crossplane/crossplane/issues/1745 but wanted to minimize the churn as much as possible in this PR